src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/custom.py

```diff
@@ -41,6 +41,59 @@ class SimpleAccessRights(Enum):
 
 
 # CUSTOM METHODS
+def iot_hub_certificate_list(client, hub_name, resource_group_name=None):
+    resource_group_name = _ensure_resource_group_name(client[0], resource_group_name, hub_name)
+    return client[1].list_by_iot_hub(resource_group_name, hub_name)
+
+
+def iot_hub_certificate_get(client, hub_name, certificate_name, resource_group_name=None):
+    resource_group_name = _ensure_resource_group_name(client[0], resource_group_name, hub_name)
+    return client[1].get(resource_group_name, hub_name, certificate_name)
+
+
+def iot_hub_certificate_create(client, hub_name, certificate_name, certificate_path, resource_group_name=None):
+    resource_group_name = _ensure_resource_group_name(client[0], resource_group_name, hub_name)
+    # Get list of certs
+    cert_list = client[1].list_by_iot_hub(resource_group_name, hub_name)
+    for cert in cert_list.value:
+        if cert.name == certificate_name:
+            raise CLIError('Certificate \'{0}\' already exists. Use \'iot hub certificate update\' to update an existing certificate.'.format(certificate_name))
```

Comment:
Would just suggest using double quotes `CLIError("...")` then you don't need to escape the single quotes and it's clearer.